pkg/agent/openflow/pipeline.go

```diff
@@ -1428,16 +1427,17 @@ func (c *client) localProbeFlow(localGatewayIPs []net.IP, category cookie.Catego
 // hostBridgeUplinkFlows generates the flows that forward traffic between the
 // bridge local port and the uplink port to support the host traffic with
 // outside. These flows are needed only on Windows Nodes.
-func (c *client) hostBridgeUplinkFlows(uplinkPort uint32, bridgeLocalPort uint32, localSubnet net.IPNet, category cookie.Category) (flows []binding.Flow) {
+func (c *client) hostBridgeUplinkFlows(localSubnet net.IPNet, category cookie.Category) (flows []binding.Flow) {
+	bridgeOFPort := uint32(config.BridgeOFPort)
 	flows = []binding.Flow{
 		c.pipeline[ClassifierTable].BuildFlow(priorityNormal).
-			MatchInPort(uplinkPort).
+			MatchInPort(config.UplinkOFPort).
 			Action().LoadRegRange(int(marksReg), markTrafficFromUplink, binding.Range{0, 15}).
 			Action().GotoTable(uplinkTable).
 			Cookie(c.cookieAllocator.Request(category).Raw()).
 			Done(),
 		c.pipeline[ClassifierTable].BuildFlow(priorityNormal).
-			MatchInPort(bridgeLocalPort).
+			MatchInPort(config.BridgeOFPort).
```

Comment:
Use the local variable `bridgeOFPort` defined in line 1431?

Reply:
The local variable bridgeOFPort is a trick to bypass the int value check for Output() (which takes an int argument). I would keep using config.BridgeOFPort for MatchInPort().

Reply:
Does it mean we could also use `config.BridgeOFPort` in line 1450, for it is already an "int" value and we don't need to convert it twice?

Reply:
No. Golang overflow check will fail because config.BridgeOFPort overflows integer.